docs/api/actions.rst

```diff
@@ -112,3 +112,21 @@ Silently
 **Aliases**: ``Quietly``
 
 .. autofunction:: Silently
+
+
+TryTo
+-----
+
+**Aliases**: ``Attempt``,
+``Attempts``,
+``AttemptsTo``,
+``AttemptTo``,
+``Either``,
+``GoesFor``,
+``GoFor``,
+``Tries``,
+``TriesT``
+``Try``
```

Comment:
Two things about this—
* In the other alias sections, the aliases are grouped by tense. I'm not sure which way is the best way, but at least the consistent way is the way i think we should go.
* It looks like you left off the "o" on `TriesTo`.

Reply:
fixed